benchmarks/cudf-merge.py

```diff
@@ -180,10 +199,39 @@ async def worker(rank, eps, args):
         pr = cProfile.Profile()
         pr.enable()
 
+    iter_results = {"bw": [], "wallclock": [], "throughput": [], "data_processed": []}
     timings = []
     t1 = clock()
-    await distributed_join(args, rank, eps, df1, df2, timings)
-    await barrier(rank, eps)
+    for i in range(args.iter):
+        iter_timings = []
+
+        iter_t = clock()
+        ret = await distributed_join(args, rank, eps, df1, df2, iter_timings)
+        await barrier(rank, eps)
+        iter_took = clock() - iter_t
+
+        # Ensure the number of matches falls within `args.frac_match` +/- 1%
+        expected_len = args.chunk_size * args.frac_match
+        expected_len_err = expected_len * 0.01
+        assert len(ret) in range(
+            expected_len - expected_len_err, expected_len + expected_len_err
+        )
```

Comment:
```suggestion
        assert abs(len(ret) - expected_len) <= expected_len_err
```
Since `expected_len_err` is a float